java-frontend/src/main/java/org/sonar/java/SonarComponents.java

```diff
@@ -193,6 +195,13 @@ public void registerTestCheckClasses(String repositoryKey, Iterable<Class<? exte
     return visitors;
   }
 
+  public List<JavaCheck> generatedCodeVisitors() {
+    return allChecks.stream()
+      .flatMap(c -> c.all().stream())
```

Comment:
c? please use a more explicit variable name. as `c.all()` is really not helping.